src/modules/task.js

```diff
@@ -1,46 +1,144 @@
+// const resetBtn = document.getElementById('reset-all');
```

Comment:
- Kindly make sure to remove this commented out line of code to make your code cleaner :recycle: